src/js/components/DataTable/doc.js

```diff
@@ -378,12 +378,26 @@ export const themeDoc = {
     }`,
     defaultValue: undefined,
   },
-
   'dataTable.header.pad': {
     description: 'The pad around the contents of the header cell.',
     type: 'string | object',
     defaultValue: undefined,
   },
+  'dataTable.resize.hover.color': {
+    description: 'The color of the resizer when hovered over.',
+    type: 'string | object',
+    defaultValue: undefined,
+  },
+  'dataTable.resize.hover.side': {
+    description: 'The side of the resizer when hovered over.',
+    type: 'string',
+    defaultValue: undefined,
+  },
+  'dataTable.resize.hover.size': {
+    description: 'The size of the resizer when hovered over.',
```

Comment:
I think we should mention that the values for this correspond to `Box.border.size` values.

Reply:
Good call.